src/sorcha/modules/PPCalculateApparentMagnitudeInFilter.py

```diff
@@ -66,7 +64,7 @@ def PPCalculateApparentMagnitudeInFilter(
     alpha = padain["Sun-Ast-Obs(deg)"].values
     H = padain[H_col].values
 
-    if LC_METHODS.get(lightcurve_choice, False):
+    if lightcurve_choice and LC_METHODS.get(lightcurve_choice, False):
```

Comment:
Extra security logic here. Since it would be possible for someone to define a subclass of `AbstractLightcurve` with a `name_id` method that returns `None`, and since `None` is a valid key in a python dictionary, we wanted to avoid any potential confusion. 

So if the default value of `lightcurve_choice` is passed in, (i.e. `None`) we will not execute this code - even if someone registers a lightcurve class with the name id `None`.